src/routes/test/+page.svelte

```diff
@@ -2767,9 +2767,24 @@
 			</div>
 		</div>
 	</section>
+	<div class="sidebar vertical">
+		<Button>Test</Button>
+	</div>
 </div>
 
 <style>
+	.sidebar {
+		border-right: 1px solid var(--colors-low);
+		padding: var(--padding);
+		width: 320px;
+		height: 100%;
+	}
+	.vertical {
+		display: flex;
+		flex-direction: column;
+		align-items: stretch;
+		gap: var(--padding);
+	}
```

Comment:
Not sure why this is split into two rules. There is also single element inside, so the `gap` does nothing and I would suggest that the border should be around the whole thing and not just on the right.